doc/Plotter.md

```diff
@@ -74,17 +76,34 @@ Class **N** represents the percentage of mapped reads that were unassigned. Sour
 - Alignments which do not overlap with any features
 
 #### Count Normalization
-Features with multiple associated classes will have their counts split evenly across these classes before being grouped and summed.
+A feature with multiple associated classes will have its counts split evenly across these classes before being grouped and summed.
 
 #### Class Chart Styles
 Proportions in rule_charts and class_charts are plotted using the same function. Styles are the same between the two. See [rule chart styles](#rule-chart-styles) for more info.
 
 
 ## replicate_scatter
-Todo
+Feature count comparisons between replicates can be used to assess the overall quality of your libraries. The replicate_scatter plot type shows these comparisons using DESeq2's normalized counts on Log<sub>2</sub> scale axes. A plot is produced for all replicate combinations in each sample group.
+
+<p float="left" align="center">
+    <img src="../images/plots/replicate_scatter.jpeg" width="80%" alt="replicate_scatter"/>
+</p>
 
 ## sample_avg_scatter_by_dge
-Todo
+Differential gene expression between sample groups can be visualized with this plot type. Normalized feature counts from DESeq2 are averaged across replicates for each sample and plotted on Log<sub>2</sub> scale axes. Features with significant expression levels will have their counts plotted with red points.
+
+<p float="left" align="center">
+    <img src="../images/plots/scatter_dge.jpeg" width="80%" alt="sample_avg_scatter_by_dge"/>
+</p>
+
+#### Customization
+The p value cutoff can be changed using the [Run Config or commandline arguments](Parameters.md#p-value). The control condition is plotted on the x-axis, but it must be specified in your Samples Sheet prior to running an end-to-end or `tiny recount` analysis. If using `tiny replot`, is not possible to change a no-control experiment to a control experiment and have these changes reflected in these plots. This is because the pipeline's DGE step must be aware of the control condition in order to perform the proper directional comparisons.
 
 ## sample_avg_scatter_by_dge_class
-Todo
\ No newline at end of file
+The previous plot type can be extended to group and color differentially expressed features by class. Classes are sorted by abundance before plotting to maximize representation.
+
+<p float="left" align="center">
+    <img src="../images/plots/scatter_dge_class.jpeg" width="80%" alt="sample_avg_scatter_by_dge_class"/>
+</p>
+
+>**Tip**: if you find that two groups of interest share proximity and are too similar in color, you can change the group's color with a modified Plot Stylesheet. The groups will be colored in the same order they are listed in the legend (not including p value outgroup), e.g. changing the color of the ERGO group means changing the 5th color in the `axes.prop_cycle` color cycler.
```

Comment:
Do you specify somewhere where to find tinyrna-light.mplstyle?  It might be worth adding a brief section describing what can be modified.  Also, as a bit of a side note, we need to replace the sample data with the simulated data so references to classes like ERGO will need to be edited at some point.

Reply:
Good idea. I've added a link to the configuration file documentation which has a short section covering the stylesheet